README.md

```diff
@@ -200,9 +200,14 @@ sudo apt install make gcc ripgrep unzip git neovim
 ```
 sudo apt update
 sudo apt install make gcc ripgrep unzip git
-echo "deb https://deb.debian.org/debian unstable main" | sudo tee -a /etc/apt/sources.list
-sudo apt update
-sudo apt install -t unstable neovim
+
+# Now we install nvim
+curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim-linux64.tar.gz
+sudo rm -rf /opt/nvim
```

Comment:
The destination is `/opt/nvim-linux64` not `/opt/nvim` is `/opt/nvim` here intentional or did you mean `/opt/nvim-linux64` ?

Reply:
I just copied it from neovim upstream, https://github.com/neovim/neovim/blob/master/INSTALL.md#pre-built-archives-2 I have no idea why they nuke that path, but it sounds like it's innocuous to me. nvim upstream should own `/opt/nvim*` as an unmodified install location. So if they want it gone if it's there, I'm cool with killing it.  All that said I left a comment on the commit https://github.com/neovim/neovim/commit/5c36701228e166f8aa89983044404ff322fe481f#r141101101

Reply:
got it, ok.